python/lsst/summit/utils/assessPeekExposure.py

```diff
@@ -0,0 +1,200 @@
+# This file is part of summit_utils.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+
+import atexit
+import logging
+import multiprocessing
+import shutil
+import time
+import warnings
+from pathlib import Path
+
+import lsst.afw.display as afwDisplay
+import matplotlib.pyplot as plt
+import numpy as np
+from astropy.table import Table
+from astropy.utils.exceptions import AstropyUserWarning
+from lsst.afw.geom import ellipses
+from lsst.daf.butler.datastore.cache_manager import DatastoreCacheManager
+from lsst.summit.utils.bestEffort import BestEffortIsr
+from lsst.summit.utils.peekExposure import PeekExposureTask
+from tqdm import tqdm
+
+# Set logger level to higher than CRITICAL to suppress all output
+silent_logger = logging.getLogger("silent_logger")
+silent_logger.setLevel(logging.CRITICAL + 1)
+
+
+# get one BestEffortIsr,afwDisplay per subprocess
+def initializer():
+    global bestEffort, pet, display, fig, ax
+    fig, ax = plt.subplots(nrows=1, ncols=1, figsize=(9, 9), facecolor="k")
+    display = afwDisplay.Display(fig, backend="matplotlib")
+    display.scale("asinh", -1, 31)
+    petConfig = PeekExposureTask.ConfigClass()
+    pet = PeekExposureTask(config=petConfig, log=silent_logger, display=display)
+    bestEffort = BestEffortIsr(embargo=True)
+
+
+# retrieve best-effort-isr and run PET on it
+def doit(idx, row, doPlot):
+    exposureId = row["exposureId"]
+    dataId = {"instrument": "LATISS", "exposure": exposureId, "detector": 0}
+    exp = bestEffort.getExposure(dataId)
+    mode = "auto"
+    donutDiameter = None
+
+    # Incorrectly labeled as photo
+    if exposureId >= 2022101200584 and exposureId <= 2022101200588:
+        mode = "donut"
+        donutDiameter = 1500
+
+    # Incorrectly labeled as giant donut
+    if exposureId >= 2022101200589 and exposureId <= 2022101201247:
+        mode = "photo"
```

Comment:
I'm not sure how the "auto" resolution is done here, but I'm guessing it's based on FITS header info; if that's the case, these should get included as corrections in [obs_lsst](https://github.com/lsst/obs_lsst/tree/main/corrections), and the (at some point) you won't need this code anymore.

Reply:
Yeah, there is a ticket already: https://jira.lsstcorp.org/browse/DM-41335?filter=13024.  I'll add a comment referencing that so we know when to delete this bit.